### PR TITLE
[RAPTOR-5872]update messages for users

### PR DESCRIPTION
**Pull request by scottp-ml**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Update a couple of the messages for users when schema validation fails.  This should make it more obvious what the issue is.

### CODE REVIEW

**Review by drenfr01 — APPROVED** [review]
LGTM

File: custom_model_runner/datarobot_drum/drum/typeschema_validation.py [hunk 1]
```diff
@@ -394,7 +394,7 @@ def validate(self, dataframe):
         passes = conditions_map[self.condition](n_columns, test_value)
         if not passes:
             return [
-                f"Num columns error, {n_columns} did not satisfy: {self.condition} {test_value}"
+                f"Num columns error, {n_columns} received but the schema says: {self.condition} {test_value}"
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/typeschema_validation.py:397]:
```suggestion
                f"Incorrect column length. {n_columns} received. However, the schema dictates that columns should be {self.condition} {test_value}"
```

File: custom_model_runner/datarobot_drum/drum/typeschema_validation.py [hunk 2]
```diff
@@ -421,7 +421,9 @@ def validate(self, dataframe):
             io_type = "Output"
 
         if any_missing and value in [missing_output_disallowed, missing_input_disallowed]:
-            return [f"{io_type} contains missing values, the task does not support missing."]
+            return [
+                f"{io_type} contains missing values, but the schema says that they are not allowed."
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/typeschema_validation.py:425]:
```suggestion
                f"{io_type} contains missing values, which the supplied schema does not allow."
```

**Review by 4dahalibut — CHANGES_REQUESTED** [review]
slightly different wording maybe

File: custom_model_runner/datarobot_drum/drum/typeschema_validation.py (hunk 1 above)
Comment (MikhailYakubovskiy) [reply custom_model_runner/datarobot_drum/drum/typeschema_validation.py:397]: How about 
f"Incorrect number of columns. {n_columns} received. However, the schema dictates that number of columns should be {self.condition} {test_value}"

File: custom_model_runner/datarobot_drum/drum/typeschema_validation.py (hunk 2 above)
Comment (MikhailYakubovskiy) [reply custom_model_runner/datarobot_drum/drum/typeschema_validation.py:425]: How about
f"{io_type} contains missing values, which the supplied task schema does not allow."

**Review by 4dahalibut — APPROVED** [review]

**Pull request merged**